src/skills-builder/skills-builder-header/SkillsBuilderHeader.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import edXLogo from '../images/edX-logo.svg';
+import messages from './messages';
+
+const SkillsBuilderHeader = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <div className="d-flex">
+      <img src={edXLogo} alt="edx-logo" className="mt-2 h-50" />
+      <div className="ml-5 vertical-line" />
+      <div className="w-100 ml-5">
+        <p className="h1 header-title">
```

Comment:
This can actually be an `<h1>` and the subheading can be an `<h2>` — mostly for accessibility

Reply:
Hey Max -- I think this advice goes against most of the advice I've had here in the past -- which is to use _styling_ like headers, but not to use header (h1/h2) components themselves.

Dumb question -- Since this is an actual page header/hero, is this why we would use an H1 here? (I'm also under the impression that there can only be a single h1 on a page for accessibility concerts)

Reply:
From what I've gathered, we dp want to use them because, without any hX elements, we have no way of telling the screen reader what the hierarchy of the content is. I wrote [this doc](https://docs.google.com/document/d/1T5Jh8Gv1kPRuwcxY82bmdCZNxTJaFoMF-RbuEwHoIyk/edit?usp=sharing) as prep for a Frontend Study Group about headings with some help from Jeff and it may clear things up.

Given that this is the Skills Builder page, I think "Skills Builder" should be the h1, or h2 at the least. And on second thought, perhaps the subtitle is fine the way you have it — it doesn't represent a section of content and therefore, doesn't require a heading element.

Reply:
OK. I'll go ahead and make this change. Thanks for the clarification. I'll make the main heading an `H1` and the subheading an `H2` -- this makes the most sense to me.